actgen/model.py

```diff
@@ -0,0 +1,23 @@
+import torch
+
+from . import nnutils
+
+def NatureDQN(input_shape=(4, 84, 84), n_actions=18):
+    shape = nnutils.get_conv2d_output_size(input_shape[1:], (8, 8), 4)
```

Comment:
should we move this class into the `actgen/agent` directory?

Reply:
I wasn't sure where to put it. It's not really an agent, it's just a neural network... Maybe we need a `models` folder?

Reply:
ahh makes sense. We could just leave it here for the moment. Maybe later we could move all the NN architectures from `nnutils` to this script, or into a `models` folder.